WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/inventory/ScanToUpdateInventoryViewModel.kt

```diff
@@ -75,6 +80,13 @@ class ScanToUpdateInventoryViewModel @Inject constructor(
             } else {
                 handleProductNotFound(status.code)
             }
+
+            tracker.track(
+                AnalyticsEvent.PRODUCT_SEARCH_VIA_SKU_SUCCESS,
+                mapOf(
+                    AnalyticsTracker.SCAN_TO_UPDATE_INVENTORY to status.code
```

Comment:
```suggestion
                    KEY_SCANNING_SOURCE to AnalyticsTracker.SCAN_TO_UPDATE_INVENTORY
```
The prop's key should be equal to "source", and value "scan_to_update_inventory"

Reply:
Sorry about that. Fixed here https://github.com/woocommerce/woocommerce-android/pull/10339/commits/4ebb6f1ad3041450a1bd4394980692d0183d7928